OpenRA.Mods.Common/Traits/Minelayer.cs

```diff
@@ -63,6 +63,12 @@ public class MinelayerInfo : TraitInfo, Requires<RearmableInfo>
 		[Desc("Ammo the minelayer consumes per mine.")]
 		public readonly int AmmoUsage = 1;
 
+		[Desc("Number of ticks it takes to lay a mine. Zero means mine-laying is instantaneous.")]
+		public readonly int LayMineTime = 0;
+
+		[Desc("Delay in ticks after mine is laid before minelayer finishes minelaying activity. Zero means no delay.")]
+		public readonly int MineLaidDelay = 20;
```

Comment:
unused variable

Could be renamed to `AfterLayingDelay` and the other variable to `PreLayDelay`